### PR TITLE
feat(core): empty path sould redirect to the console page

### PR DESCRIPTION
**Pull request by simeng-li**
## Summary

Empty path should redirect to the console page.

Rename the koa-client-session-guard to koa-proxy-guard. 
Add console redirect if empty path. 

## Linear Issue Reference

log-2597

## Testing

test locally
UT passed
@logto-io/eng

### CODE REVIEW

File: packages/core/src/middleware/koa-proxy-guard.ts [hunk 1]
```diff
@@ -18,18 +18,26 @@ export default function koaSpaSessionGuard<
   return async (ctx, next) => {
     const requestPath = ctx.request.path;
     const packagesPath = fromRoot ? 'packages/' : '..';
-    const distPath = path.join(packagesPath, 'ui', 'dist');
+    const clientPath = path.join(packagesPath, 'ui', 'dist');
 
-    // Guard client routes only
+    // Empty path Redirect
+    if (requestPath === '/') {
+      console.log(requestPath);
```
Comment (gao-sun) [inline packages/core/src/middleware/koa-proxy-guard.ts:25]: unnecessary log?

File: packages/core/src/middleware/koa-proxy-guard.test.ts [hunk 2]
```diff
@@ -3,7 +3,7 @@ import { Provider } from 'oidc-provider';
 import { MountedApps } from '@/env-set';
 import { createContextWithRouteParameters } from '@/utils/test-utils';
 
-import koaClientSessionGuard, { sessionNotFoundPath } from './koa-client-session-guard';
+import koaClientSessionGuard, { sessionNotFoundPath } from './koa-proxy-guard';
```
Comment (gao-sun) [inline packages/core/src/middleware/koa-proxy-guard.test.ts:6]: forgot changing the import name?

File: packages/core/src/middleware/koa-proxy-guard.ts (hunk 1 above)
Comment (simeng-li) [reply packages/core/src/middleware/koa-proxy-guard.ts]: yep just about to remove

File: packages/core/src/middleware/koa-proxy-guard.test.ts (hunk 2 above)
Comment (simeng-li) [reply packages/core/src/middleware/koa-proxy-guard.test.ts]: yep

**Review by IceHe — APPROVED** [review]
LGTM

**Pull request merged**